### PR TITLE
Add steps of CA rotation for api.ci

### PR DESCRIPTION
**Pull request by hongkailiu**
/cc @openshift/openshift-team-developer-productivity-test-platform 

WIP

/hold

### CODE REVIEW

File: core-services/cluster/ca-cert_README.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+```bash
+### The new cert is in the attachment of the ticket; Download them
+$ ls *api*.zip
+api_ci_openshift_org_8015712.zip origin-ci-ig-m-428p.api.zip
+### unzip 
+$ cd api_ci_openshift_org_8015712
+$ ls ./*.crt
+api_ci_openshift_org_8015712/api_ci_openshift_org.crt api_ci_openshift_org_8015712/DigiCertCA.crt
+$ cat api_ci_openshift_org.crt DigiCertCA.crt > ssl.crt 
+### Clayton: this is a bug of openssl: https://coreos.slack.com/archives/CBN38N3MW/p1575316749423100?thread_ts=1575294657.383300&cid=CBN38N3MW
```
Comment (droslean) [inline core-services/cluster/ca-cert_README.md]: I don't think its a good idea to add a comment with a slack thread here. I would prefer to have an issue open and comment that instead.

File: core-services/cluster/ca-cert_README.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+$ openssl verify ssl.crt 
+C = US, ST = North Carolina, L = Raleigh, O = "Red Hat, Inc.", OU = RHC Cloud Operations, CN = api.ci.openshift.org
+error 20 at 0 depth lookup: unable to get local issuer certificate
+error ssl.crt: verification failed
+
+### the keys are added into the tickets
+###https://coreos.slack.com/archives/CCX9DB894/p1575926069119000?thread_ts=1575300232.249000&cid=CCX9DB894
```
Comment (droslean) [inline core-services/cluster/ca-cert_README.md]: here too..

File: core-services/cluster/ca-cert_README.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+$ ls *.key
+wildcard.svc.ci.openshift.org.key api.ci.openshift.org.key
+```
+
+### back up existing certs:
+
+```bash
+$ for master in origin-ci-ig-m-428p origin-ci-ig-m-f3g1 origin-ci-ig-m-pbj3; do gcloud compute ssh "${master}" -- "sudo tar -zcvf /root/master.bk.20191202.tar.gz /etc/origin/master"; done
```
Comment (droslean) [inline core-services/cluster/ca-cert_README.md]: no need to add `sudo` here.

Comment (hongkailiu) [reply core-services/cluster/ca-cert_README.md]: This is what I got ...

```bash
$ gcloud compute ssh "origin-ci-ig-m-428p" -- "ls /etc/origin/master"
ls: cannot access /etc/origin/master: Permission denied
```

File: core-services/cluster/ca-cert_README.md (hunk 2 above)
Comment (hongkailiu) [reply core-services/cluster/ca-cert_README.md]: Removed.

File: core-services/cluster/ca-cert_README.md (hunk 1 above)
Comment (hongkailiu) [reply core-services/cluster/ca-cert_README.md]: Removed.

File: core-services/cluster/ca-cert_README.md (hunk 3 above)
Comment (droslean) [reply core-services/cluster/ca-cert_README.md]: Ah yes. I though gcloud ssh connects you as root, but it seems no.

**Pull request merged**